ffv1.md

```diff
@@ -1099,23 +1107,25 @@ If state_transition_delta is not present in the FFV1 bitstream, all Range coder
 
 ### colorspace_type
 
-`colorspace_type` specifies the color space.
+`colorspace_type` specifies the Pixel transformation used by the encoder, as well as interleave method.
```

Comment:
The act of encoding is generally not normative. Thats how all ITU/ISO video specs are written IIRC. I dont think we should imply too much about the encoder as this would make it very ambigous if a encoder doing something different was compliant.